bin/.travis/init_solr.sh

```diff
@@ -239,21 +237,6 @@ add_core() {
     fi
 }
 
-post_documents() {
-    dir_name=$1
-    solr_docs=$2
-    solr_core=$3
-    solr_port=$4
-      # Post documents
-    if [ -z "${solr_docs}" ]
-    then
-        echo "SOLR_DOCS not defined, skipping initial indexing"
-    else
-        echo "Indexing $solr_docs"
-        java -Dtype=application/json -Durl=http://localhost:$solr_port/solr/$solr_core/update/json -jar $dir_name/example/exampledocs/post.jar $solr_docs
-    fi
-}
-
 check_version() {
     case $1 in
         3.5.0|3.6.0|3.6.1|3.6.2|4.0.0|4.1.0|4.2.0|4.2.1|4.3.1|4.4.0|4.5.0|4.5.1|4.6.0|4.6.1|4.7.0|4.7.1|4.7.2|4.8.0|4.8.1|4.9.0|4.9.1|4.10.0|4.10.1|4.10.2|4.10.3);;
```

Comment:
Yes, but I'm also using your fork of moliware's Solr script to set up single core, need to put this in the our local script as well.